fidesctl/src/fidesctl/cli/__init__.py

```diff
@@ -78,6 +81,33 @@ def cli(ctx: click.Context, config_path: str, local: bool) -> None:
     if not ctx.invoked_subcommand:
         click.echo(cli.get_help(ctx))
 
+    if (
+        ctx.obj["CONFIG"].user.analytics_opt_out is None
+        and ctx.invoked_subcommand != "init"
+    ):
+        ctx.obj["CONFIG"].user.analytics_opt_out = bool(
+            input(OPT_OUT_COPY).lower() == "n"
+        )
+
+        config_updates = {
+            "cli": {"analytics_id": ctx.obj["CONFIG"].cli.analytics_id},
+            "user": {"analytics_opt_out": ctx.obj["CONFIG"].user.analytics_opt_out},
+        }
+
+        try:
+            update_config_file(config_updates)
```

Comment:
This flow is interesting

Reply:
Interesting for sure 😂 

We basically had to settle on the idea that `init` is a bit of a special case (opening another issue/PR around this later today) but it was one the many chicken/egg scenarios we hit. We want to trigger the opt out routine for any command where the user hasn't opted in/out yet, but it would also flag the init command when they were going to store the config for the first time. This allowed us to have both methods for getting consent work together without colliding